src/Classes/Options/Particles/Particles.ts

```diff
@@ -76,6 +94,23 @@ export class Particles implements IParticles {
             this.shape.load(data.shape);
             this.size.load(data.size);
             this.shadow.load(data.shadow);
+
+            let strokeToLoad = data.stroke ?? data.shape?.stroke;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'strokeToLoad' is never reassigned; use 'const' instead of 'let'.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5274115)